preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -48,10 +49,21 @@
   private PreferencesFx(Class<?> saveClass, Category... categories) {
     // asciidoctor Documentation - tag::testMock[]
     preferencesFxModel = new PreferencesFxModel(
-        new StorageHandler(saveClass), new SearchHandler(), new History(), categories
+        new DefaultStorageHandler(saveClass), new SearchHandler(), new History(), categories
     );
     // asciidoctor Documentation - end::testMock[]
+  }
+
+  private PreferencesFx(AbstractStorageHandler storageHandler, Category... categories) {
```

Comment:
It would be better IMHO to open this class for extension, otherwise we'll end up passing multiple arguments to the constructor. See #21 for the beginning of this discussion.

Reply:
I would like to leave this as is for now (because im waiting for this PR to be merged to able finish my little project).
It can be refactored later in another PR for the #21